lib/worker.js

```diff
@@ -292,40 +292,31 @@ class AssetComputeWorker {
         }
 
         // post-processing // TODO: Refactor this to share partas with post-processing in `processRendition` func
+        // eslint-disable-next-line prefer-const
```

Comment:
It should be OK to use `const rendition`, I don't think we reassign it.

Reply:
I had to change the usage back to `let rendition` as inside `doPostProcess` we actually modify `rendition` so not returning the updated code seems deceiving.

Reply:
You mean because the reference changes at this line: https://github.com/adobe/asset-compute-sdk/blob/3463b14a66469f4e97f2a1cfc5816f978e8cd31d/lib/worker.js#L349 ?

Reply:
as we update the `rendition` here, for the new `rendition` to take effect after the method returns, it required to be reassigned https://github.com/adobe/asset-compute-sdk/blob/269770a37a8f9d1a65ea35bf96216d40997330bb/lib/worker.js#L226.

Reply:
So we need the new rendition only for the log statement? Because in that case you could get the name before the call in a variable. So you could remove the need to return a rendition too.

Reply:
actually we also needed changed `rendition` for https://github.com/adobe/asset-compute-sdk/blob/269770a37a8f9d1a65ea35bf96216d40997330bb/lib/worker.js#L242
which otherwise will use `rendition0.jpg` instead of `post-rendition0.jpg`